In filter with node ids (#1991)
cognitedata/cognite-sdk-python: CHANGELOG.md

- `Fixed` for any bug fixes.
- `Security` in case of vulnerabilities.

## [7.63.10] - 2024-10-22
### Added

> - Data modeling: When using the `In` filter to filter on direct relations, `NodeId`s can now be used directly as values.

Reviewer: ```suggestion
- Data modeling filters now support the use of `NodeId` (and `EdgeId`) directly.
```
